src/src/hooks/Capabilities.js

```diff
@@ -290,3 +290,53 @@ export function useCapabilityMembersApplications(capabilityDefinition) {
     membersApplicationsList,
   };
 }
+
+export function useCapabilityMetadata(capabilityDefinition) {
+  const { responseData, sendRequest: sendGetJsonMetadataRequest } =
+    useSelfServiceRequest();
+  const { sendRequest: sendSetJsonMetadataRequest } = useSelfServiceRequest();
+  const [isLoadedMetadata, setIsLoadedMetadata] = useState(false);
+  const [metadata, setMetadata] = useState(null);
+
+  const link = capabilityDefinition?._links?.metadata;
+
+  useEffect(() => {
+    if (link && (link.allow || []).includes("PUT")) {
+      sendGetJsonMetadataRequest({
+        urlSegments: [link.href],
+        method: "GET",
+      }).then((_) => {});
```

Comment:
Why this `.then()` block?
I believe this forces synchronicity into our asynchronous flow.
If this is desired, then using `await` is cleaner.
And doing nothing with the result makes the synchronicity a bit futile.

However, simply monitoring changes to `responseData` should give us everything we need here?

Reply:
it was to get rid of a warning: have changed it to await plus async

Reply:
changed to void to ignore returned value